makefile

```diff
@@ -33,6 +33,7 @@ build-docker:
 
 tests:
 	go test -race -timeout 30s -cover ./cmd ./check
```

Comment:
I think this line is redundant. In the line below you are running tests in all packages, including ./cmd and .check.

Reply:
Done